### ISSUE TITLE
Need less opinionated backend

### ISSUE THREAD

**Issue by pkese** [foxhound87/rfx-stack#8]
As much as I like this starter kit, I think it would have much better success if it was a bit less opinionated (or more configurable) on the backend.

I think there are a lot of people who would benefit from a clean react+mobx+react-router stack like this.

But then there is just 0.0{something} percent of people who would use that stack with Feathers.js and out of that just a 0.{something} percent of people who would use Feathers.js specifically with MongoDB instead of one of other 20 database adapters that Feathers.js supports.

Even though I think Feathers.js is really cool (and I didn't even know about it before) and that Feathers does benefit from some promotion, I sadly don't consider MongoDB cool at all. 

Since Feathers comes with some really streamlined command line tools to quickly kickstart a backend project with a database of one's choice, I think this project would gain much more popularity, if instead of shipping with fully opinionated backend code, there would be just some written (and ideally copy-pastable) instructions on how to kickstart a Feathers backend yourself.

The written instructions could be for MongoDB instructions no problem. It is just that for non-mongodb people it would be much easier to start a project from scratch rather than first figuring out which parts of existing code they should first delete before new stuff can be implemented.

Please don't consider this as a negative opinion. I really care about this starter and find it as a awesome selection of tools an libraries and I wish it would gain more popularity. I feel that it has great potential in the mobx space but that it might miss its goals just because it goes a little bit too far.

One option would be to split the project into client (react) and server (feathers) code and have an option for other people to contribute different backend solutions. I myself might be willing to write one for Postgres or Rethinkdb.

**Comment by foxhound87** [comment]
@pkese Thank you for your overview.

Originally I created this starter to experiment React, Webpack, MobX, and than I've thrown in FeathersJS for a fast-prototyping for others functionalities (such as server side rendering) and provide a demo data to show MobX stores in action.

But I understand your point of view and already was in my plan to evolve this repo in something more flexible. I agree with you about splitting the project into more packages, in fact, right now, the project is almost monolithic and rigid, and would be nice to make this more suitable for microservices development.

The main principles I'd like to keep by which I created this stack are: simplicity and a completely reactive interface: so MobX instead the more complex Redux for state management and a real-time backend like Feathers that was simple to implement which massively reduces the boilerplate for a socket.io app.

I have other ideas too, if you or other people are interested to contribute I will setup a slack chat.
